data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-03', 2, true, 8);
+INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals VALUES (4, 'Devimon', '2017-05-23', 5, true, 11);
```

Comment:
* Kindly correct the date of birth for these animals ( Gabumon, Devimon) as they are not correct 
![image](https://github.com/zunairkhan811/Vet-clinic/assets/52497246/57e45ae2-93cb-4267-b908-86afd1918105)

Reply:
-Thanks for the suggestions. Your suggestion has been implemented.